### PR TITLE
[Frontend][Darknet] L2 normalization support in darknet

### PR DESCRIPTION
**Pull request by PariksheetPinjari909**
Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from others in the community.

### CODE REVIEW

**Comment by PariksheetPinjari909** [comment]
@siju-samuel @yzhliu Welcome for review.

File: nnvm/python/nnvm/frontend/darknet.py [hunk 1]
```diff
@@ -267,6 +267,13 @@ def _darknet_upsampling(inputs, attrs):
     new_attrs['scale'] = attrs.get('scale', 1)
     return _darknet_get_nnvm_op(op_name)(*inputs, **new_attrs), None
 
+def _darknet_l2normalize(inputs, attrs):
+    """Process the l2 normalization operation."""
+    op_name, new_attrs = 'l2_normalize', {}
+    new_attrs['eps'] = attrs.get('eps', 0)
+    new_attrs['axis'] = attrs.get('axis', 1)
```
Comment (yzhliu) [inline nnvm/python/nnvm/frontend/darknet.py]: I don't really know darknet, would appreciate if you could point out where these two attrs come from. The overall functionality looks good to me.

Comment (PariksheetPinjari909) [reply nnvm/python/nnvm/frontend/darknet.py]: Sure. The darknet computes the l2 normalization always along the channel axis. So it doesn't store the axis information anywhere. And while dividing by sum it doesn't check if sum is greater than eps or not. 
So i have made default eps as 0 and axis as 1 to make it compatible with TVM's L2 normalization implementation. You can have look at darknet's l2 normalization implementation [here](https://github.com/pjreddie/darknet/blob/61c9d02ec461e30d55762ec7669d6a1d3c356fb2/src/blas.c#L126)

Comment (yzhliu) [reply nnvm/python/nnvm/frontend/darknet.py]: got it. thanks!

**Pull request merged**